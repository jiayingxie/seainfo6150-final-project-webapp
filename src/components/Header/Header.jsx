import React, { Component, Fragment } from 'react'
import styles from './Header.module.css';

export default class Header extends Component {
    render() {
        return (
            <Fragment >
                <div className={styles.container}>
                    <span className={styles.neu}> Northeastern University </span><hr />
                    <span className={styles.coe}>COLLEGE OF ENGINEERING</span>
                </div>
            </Fragment>
        )
    }
}
