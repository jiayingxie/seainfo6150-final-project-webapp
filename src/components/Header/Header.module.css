.container {
    display: flex;
    flex-direction: column;
    align-items: center;
    background-color: rgb(123, 181, 240);
}

.neu {
    font-family: cursive, sans-serif;
    color: #c00;
    font-weight: bold;
    font-size: 1.5rem;
}

.coe {
    font-family: Georgia, Times, sans-serif;
    font-weight: bold;
    font-size: 1rem;
}
