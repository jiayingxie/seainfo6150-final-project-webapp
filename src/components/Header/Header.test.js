import React from "react";
import Header from "./Header";

describe("Header tests", () => {
    it("renders correctly", () => {
        const { container } = render(<Header />);
        expect(container).toMatchSnapshot();
    });
});