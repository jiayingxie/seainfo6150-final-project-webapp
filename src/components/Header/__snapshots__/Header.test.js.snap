// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header tests renders correctly 1`] = `
<div>
  <div
    class="container"
  >
    <span
      class="neu"
    >
       Northeastern University 
    </span>
    <hr />
    <span
      class="coe"
    >
      COLLEGE OF ENGINEERING
    </span>
  </div>
</div>
`;
